platform/android/os_android.cpp

```diff
@@ -514,6 +515,7 @@ void OS_Android::process_touch(int p_what, int p_pointer, const Vector<TouchPos>
 					ev.screen_touch.pressed = false;
 					ev.screen_touch.x = touch[i].pos.x;
 					ev.screen_touch.y = touch[i].pos.y;
+					input->set_mouse_pos(Point2(touch[i].pos.x,touch[i].pos.y));
```

Comment:
Actually, sorry, didn't check it in full before.... Shouldn't this be just before the loop (on :507)? (using `git commit --amend` would auto-squash the change)